.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build, Push, Deploy
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  build-push:
+    name: Build docker image
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Set current date as env variable"
+        run: |
+          echo "builddate=$(date +'%Y.%m.%d-%H-%M-%S')" >> $GITHUB_OUTPUT
+        id: version  # this is used on variable path
+
+      - name: Check out the repo
+        uses: actions/checkout@v6
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Log in our registry
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: DockerfileInsta
+          push: true
+          tags: |
+            chillerdragon/ddnet-insta:${{ github.event.release.tag_name }},
+            chillerdragon/ddnet-insta:latest
```

Comment:
I don't like my personal name there. If this reflects official releases there should be a ddnet-insta docker account. I will look into creating one.